flink-python/pyflink/datastream/formats/csv.py

```diff
@@ -131,7 +131,7 @@ def add_columns_from(self, schema: 'CsvSchema') -> 'CsvSchemaBuilder':
         :param schema: Another :class:`CsvSchema`.
         """
         self._j_schema_builder.addColumnsFrom(schema._j_schema)
-        for field in cast(schema._data_type, RowType):
```

Comment:
Could we update the test cases to cover this method?

Reply:
I could add this in the CSV PR.